everyvoice/_version.py

```diff
@@ -1,5 +1,5 @@
-# [Valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#versioning)
-# [PEP 440 – Version Identification and Dependency Specification](https://peps.python.org/pep-0440/)
-# [Specifying Your Project’s Version](https://setuptools.pypa.io/en/latest/userguide/distribution.html)
+# [Valid version numbers](https://packaging.python.org/en/latest/discussions/versioning/#versioning)  # noqa: E501
+# [PEP 440 – Version Identification and Dependency Specification](https://peps.python.org/pep-0440/)  # noqa: E501
+# [Specifying Your Project’s Version](https://setuptools.pypa.io/en/latest/userguide/distribution.html)  # noqa: E501
 # [N!]N(.N)*[{a|b|rc}N][.postN][.devN]
-VERSION = "0.2.0a0"
+VERSION = "0.3.0a0"
```

Comment:
We didn't actually publish 0.2.0a0 yet, should stay 0.2.0a0 as the version we're currently developing until we actually publish it.